### ISSUE TITLE
how to generate win rates and affinities

### ISSUE THREAD

**Issue by GuoleiSun** [StanfordVL/taskonomy#36]
Hi, thanks for the excellent work.

I have tried your models and would like to generate the win rates and affinities. In taskonomy/code/README.md, it says that "Generate win rates and affinities with one of the methods in analysis/", but there is no "analysis" folder. It also says that taskonomy/code has the following directory. But there is no "notebooks" folder.

![Screenshot from 2020-10-13 11-11-12](https://user-images.githubusercontent.com/22496918/95840766-dbb5e400-0d44-11eb-96a9-6303a5637e87.png)

Could you please add that?

Thanks

**Comment by amir32002** [comment]
@GuoleiSun did you see this https://github.com/StanfordVL/taskonomy/tree/master/results?

**Comment by GuoleiSun** [comment]
Yes, I saw this. But I think it only contains the results and there are no codes to generate the win rates and affinities.

See the screenshots below.
![Screenshot from 2020-10-16 16-54-09](https://user-images.githubusercontent.com/22496918/96274122-75dd8c80-0fd0-11eb-9b5e-2246e3a80869.png)
![Screenshot from 2020-10-16 16-54-29](https://user-images.githubusercontent.com/22496918/96274133-783fe680-0fd0-11eb-8f99-e0ffe6570980.png)
